### ISSUE TITLE
Export CSV: negative numbers are corrupted

### ISSUE THREAD

**Issue by Jamim** [grafana/grafana#18177]
**What happened**:
Negative numbers in exported CSV are corrupted due to the value escaping that was introduced in #17363.
```
"Time";"Metric";"Value"
"2019-07-16 19:18:51";"A-series";"1.00"
"2019-07-16 19:19:51";"A-series";"'-2.00"
```
As you can see, `-2` was exported as `"'-2.00"` with a single quote preceding the value.
This value could not be processed properly by spreadsheet software.

**What you expected to happen**:
All exported numbers can be properly processed by spreadsheet software.
```
"Time";"Metric";"Value"
"2019-07-16 19:18:51";"A-series";1
"2019-07-16 19:19:51";"A-series";-2
```

**How to reproduce it (as minimally and precisely as possible)**:

1. Add the `TestData DB` data source
2. Import the [Test dashboard](https://gist.githubusercontent.com/Jamim/befd3c825424ab31580ffe7509673f0d/raw/440fea4cb9009070d12de62b53cf72c963a3d538/test-dashboard.json)
3. For the `Test data` table do `Export CSV`

<details>
<summary>Screenshots</summary>

![add-datasource-1](https://user-images.githubusercontent.com/5607572/61460794-272de680-a978-11e9-87da-e4fd530155fa.png)

![add-datasource-2](https://user-images.githubusercontent.com/5607572/61460814-30b74e80-a978-11e9-9a56-59f76998e3bf.png)

![import-dashboard-1](https://user-images.githubusercontent.com/5607572/61460889-58a6b200-a978-11e9-8bc4-49d0fa5449c4.png)

![import-dashboard-2](https://user-images.githubusercontent.com/5607572/61460904-5e03fc80-a978-11e9-9759-754f4922c26e.png)

![import-dashboard-3](https://user-images.githubusercontent.com/5607572/61460915-62301a00-a978-11e9-95e5-d160a975d220.png)

![export-csv](https://user-images.githubusercontent.com/5607572/61460921-678d6480-a978-11e9-9ef8-d8243ec13a7a.png)

</details>

**Environment**:
- Grafana version: `v6.2.4` (ee01897), `v6.4.0-pre` (0ef4060b9)
- Data source type & version: any that could give you some table data
- OS Grafana is installed on: `Linux`
- User OS & Browser: any
- Grafana plugins: none

**Comment by fmgonsalves** [comment]
Hey guys,

Where are we at on this?  Running into the same problem at the moment.

**Comment by pcsegal** [comment]
I'm also having problems with this. Some users need to export table data to CSV and import it in Excel, and this issue causes difficulties for them.

**Comment by torkelo** [comment]
Looks like this is fixed in 7.1 at least
